scripts/detect_errors.py

```diff
@@ -4,7 +4,9 @@
 import sys
 
 def format_msg(msg):
-    return f"{msg['file_name']}:{msg.get('pos_line')}:{msg.get('pos_col')}: {msg.get('severity')}: {msg.get('text')}\n"
+    # Encoded for https://github.com/actions/toolkit/blob/master/docs/commands.md#log-level
+    msg_text = msg.get('text').replace("\n", "%0A")
+    return f"::{msg.get('severity')} file={msg['file_name']},line={msg.get('pos_line')},col={msg.get('pos_col')}::{msg_text}"
```

Comment:
Would it be possible to add the filename, line and column to `msg_text` and get the best of both worlds?

Reply:
Yeah, that sounds fine to me

Reply:
Done